src/html/index.html

```diff
@@ -162,7 +162,7 @@ <h3>Name: {{ stream.video_and_stream.name }}</h3>
                 },
                 checkResponse: async function(response) {
                     if (!response.ok) {
-                        const text = await response.text().replaceAll("\\n", "\n").replaceAll("\\\"", "\"")
+                        const text = await response.text().then((text) => text.replace(/^"(.+(?="$))"$/, '$1').replaceAll("\\n", "\n").replaceAll("\\\"", "\""))
```

Comment:
Can you be a bit more clear about what this regex is replacing ?

Reply:
In sequence:
1. get content from inside the external double quotes pair
2. unescape new lines
3. unescape double quotes

Reply:
Ok, I would recommend to add a comment for the first regex and a minimal example of what it should match and what is the content of text just for the clarity. A look head/behind regex is never trivial.

Reply:
that's fair

Reply:
@patrickelectric done